modules/nw-ovn-kubernetes-examine-sb-database-contents-ref.adoc

```diff
@@ -8,6 +8,11 @@
 
 The following table describes the command line arguments that can be used with `ovn-sbctl` to examine the contents of the southbound database.
 
+[NOTE]
+====
+Open a remote shell in the pod you wish to view the contents of and then run the `ovn-sbctl` commands.
```

Comment:
ditto which pod

Reply:
I placed this note in the docs because each pod will have different outputs for the commands that follow. Maybe this is understood or too basic. I'm happy to remove I just thought it could help the user to know that they should first rsh into the pod they want to view the contents of before executing the commands.

Happy to take it out. Let me know what you think @tssurya

Reply:
its fine, let's leave it as is. resolving comment.